app/ui-message/client/messageBox/userActionIndicator.js

```diff
@@ -0,0 +1,53 @@
+import { Template } from 'meteor/templating';
+import _ from 'underscore';
+
+import { UserAction } from '../../../ui';
+import { t } from '../../../utils';
+import { getConfig } from '../../../ui-utils/client/config';
+
+import './userActionIndicator.html';
+
+const maxUsernames = parseInt(getConfig('max-usernames-typing')) || 2;
+
+Template.userActionIndicator.helpers({
+	data() {
+		const roomAction = UserAction.get(this.tmid || this.rid);
+		if (_.isEmpty(roomAction)) {
+			return [];
+		}
+		const userActions = [];
+		const activities = Object.keys(roomAction) || {};
+		activities.forEach((key) => {
```

Comment:
Using a map you make the code easier to understand
```suggestion
		const userActions = activities.map((key) => {
```